functions/ocr/app/main.py

```diff
@@ -31,7 +33,19 @@
 
 
 # [START functions_ocr_detect]
-def detect_text(bucket, filename):
+def detect_text(bucket: str, filename: str) -> None:
+    """
+    Extract the text from an image uploaded to Cloud Storage, then
```

Comment:
A one line summary followed by a blank line and then more details is preferred. But it's not required.